db/migrate/20150512083842_remove_old_translation_from_listing_unit.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveOldTranslationFromListingUnit < ActiveRecord::Migration
+  def up
+    remove_column :listing_units, :translation_key
+  end
+
+  def down
+    add_column :listing_units, :translation_key, :string, limit: 64, after: :quantity_selector
+  end
+end
```

Comment:
If I understood the blog post correctly, this https://github.com/sharetribe/sharetribe/commit/f7f3e68bf4617ffc054b76938a406f13a2348a22 should do it?